crates/diesel_models/src/business_profile.rs

```diff
@@ -84,6 +86,84 @@ pub struct BusinessProfileUpdateInternal {
     pub payment_link_config: Option<serde_json::Value>,
     pub session_expiry: Option<i64>,
     pub authentication_connector_details: Option<serde_json::Value>,
+    pub is_extended_card_info_enabled: Option<bool>,
+}
+
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
+pub enum BusinessProfileUpdate {
+    Update {
+        profile_name: Option<String>,
+        modified_at: Option<time::PrimitiveDateTime>,
+        return_url: Option<String>,
+        enable_payment_response_hash: Option<bool>,
+        payment_response_hash_key: Option<String>,
+        redirect_to_merchant_with_http_post: Option<bool>,
+        webhook_details: Option<serde_json::Value>,
+        metadata: Option<pii::SecretSerdeValue>,
+        routing_algorithm: Option<serde_json::Value>,
+        intent_fulfillment_time: Option<i64>,
+        frm_routing_algorithm: Option<serde_json::Value>,
+        payout_routing_algorithm: Option<serde_json::Value>,
+        is_recon_enabled: Option<bool>,
+        applepay_verified_domains: Option<Vec<String>>,
+        payment_link_config: Option<serde_json::Value>,
+        session_expiry: Option<i64>,
+        authentication_connector_details: Option<serde_json::Value>,
+    },
+    ExtendedCardInfoUpdate {
+        is_extended_card_info_enabled: Option<bool>,
```

Comment:
Might as well make it mandatory, since it would be a no-op if the parameter wasn't given anyways. Not a blocker tho.
```suggestion
        is_extended_card_info_enabled: bool,
```